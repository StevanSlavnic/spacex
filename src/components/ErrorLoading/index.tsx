export default function ErrorLoading() {
  return (
    <div>
      <p>Something went wrong...</p>
      <p>Try refreshing the page or check your internet connection.</p>
    </div>
  );
}
