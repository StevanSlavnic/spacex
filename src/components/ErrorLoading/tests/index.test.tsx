import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import ErrorLoading from "..";

describe("ErrorLoading", () => {
  it("renders the error message", () => {
    const { container, getByText } = render(<ErrorLoading />);
    expect(getByText("Something went wrong...")).toBeInTheDocument();
    expect(
      getByText("Try refreshing the page or check your internet connection.")
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
