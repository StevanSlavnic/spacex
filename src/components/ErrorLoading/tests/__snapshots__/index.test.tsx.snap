// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorLoading renders the error message 1`] = `
<div>
  <div>
    <p>
      Something went wrong...
    </p>
    <p>
      Try refreshing the page or check your internet connection.
    </p>
  </div>
</div>
`;
